test/servinge2e/kourier/servicemesh_test.go

```diff
@@ -72,8 +72,6 @@ func setupNamespaceForServiceMesh(ctx *test.Context, serviceMeshNamespace, testN
 	test.CreateServiceMeshMemberRollV1(ctx, test.ServiceMeshMemberRollV1("default", serviceMeshNamespace, testNamespace))
 
 	test.CreateNetworkPolicy(ctx, test.AllowFromServingSystemNamespaceNetworkPolicy(testNamespace))
-	test.LabelNamespace(ctx, "knative-serving", test.KnativeSystemNamespaceKey, "true")
```

Comment:
We dont want to verify `socommon.ServerlessCommonLabelKey`?

Reply:
No, `test.LabelNamespace` does not verify the label but just added the manual label for the NetworkPolicy. 
We don't need to add the label anymore. I removed `LabelNamespace` helper function as well.

Reply:
But we do add a label in the namespace resources we dont want an e2e test for that?